docs/resources/elastigroup_gcp.md

```diff
@@ -18,6 +18,7 @@ resource "spotinst_elastigroup_gcp" "example" {
   description     = "spotinst gcp group"
   service_account = "example@myProject.iam.gservicecct.com"
   startup_script  = ""
+  instance_name_prefix = "test"
```

Comment:
give a name with combination of alphanumeric which is accepted

Reply:
Updated